lib/elixir/test/elixir/inspect/algebra_test.exs

```diff
@@ -258,19 +245,16 @@ defmodule Inspect.AlgebraTest do
     assert force_unfit("ab") == {:doc_force, "ab"}
     assert force_unfit(empty()) == {:doc_force, empty()}
 
-    # Consistent with definitions
-    assert next_break_fits("ab") == {:doc_fits, "ab", :enabled}
-    assert next_break_fits(empty()) == {:doc_fits, empty(), :enabled}
-    assert next_break_fits("ab", :disabled) == {:doc_fits, "ab", :disabled}
-    assert next_break_fits(empty(), :disabled) == {:doc_fits, empty(), :disabled}
-
     # Consistent formatting
-    doc = force_unfit(concat(glue(glue(glue("hello", "a"), "b"), "c"), "d"))
-    assert render(doc, 20) == "hello\na\nb\ncd"
-    assert render(next_break_fits(doc, :enabled), 20) == "hello a b cd"
```

Comment:
I think these old tests don't work anymore when trying the deprecated implementation relying on`:optimistic`:

```elixir
    code:  assert render(next_break_fits(doc, :enabled), 20) == "hello a b cd"
     left:  "hello\na\nb\ncd"
     right: "hello a b cd"
```

Is this expected or could be a regression?

Reply:
Fixed!